app/controllers/oauth_controller.rb

```diff
@@ -4,21 +4,14 @@
 class OauthController < ApplicationController
   # OAuth2 authentication process:
   # 1. Construct the authorize URL to ISSO with required parameters, such as,
-  # client ID, client secret, redirect URI, state, and scope
+  # client ID, client secret, redirect URI, state, and scope.
   # 2. Redirect the user to the authorize URL for logging in
   # 3. After logged in, redirect the user back to our application.
   # The redirect URL should now come with an authorization code.
   # 4. Get the acceess token with the authorization code by requesting to the access token URL of ISSO.
   # And redirect back to the redirect URI again.
-  # 5. Includes the access token we just got in the header for making HTTP requests.
   OAUTH_CLIENT = OAuth2::Client.new(ENV['CLIENT_ID'], ENV['CLIENT_SECRET'], site: ENV['OAUTH_SITE'])
-  OAUTH_SCOPE  = 'login:staff'
-
-  class << self
-    # token is the initalized ACCESS_TOKEN class after successfully called OAUTH_CLIENT.auth_code.get_token
-    # in callback method
-    attr_accessor :token
-  end
+  OAUTH_SCOPE  = 'openid login:staff offline_access'
```

Comment:
Change the scope here for getting the JWT access token, which includes the user's info such as email address.

Reply:
Excellent. That's going to come in handy.

Reply:
Thanks!!!